package.json

```diff
@@ -1,38 +1,34 @@
 {
   "name": "@webstudio-is/cli",
-  "version": "0.3.0",
+  "version": "0.4.0",
   "description": "Webstudio CLI",
   "author": "Webstudio <github@webstudio.is>",
   "repository": "git@github.com:webstudio-is/webstudio-cli.git",
   "homepage": "https://webstudio.is",
   "scripts": {
-    "build": "yarn clean && esbuild ./src/*  --outdir=./lib --platform=node",
-    "clean": "rm -fr ./lib",
-    "typecheck": "tsc"
-  },
-  "devDependencies": {
-    "@types/fs-extra": "^9.0.13",
-    "@types/node": "^17.0.21",
-    "@types/node-fetch": "^2.6.1",
-    "esbuild": "^0.14.25",
-    "typescript": "^4.6.2"
-  },
-  "dependencies": {
-    "@webstudio-is/http-client": "^0.54.0",
-    "fs-extra": "^10.0.1",
-    "inquirer": "^8.2.1",
-    "meow": "^10.1.2",
-    "node-fetch": "^3.2.2"
+    "dev": "pnpm build --watch",
+    "prebuild": "tsx ./lib/prebuild.js",
+    "build": "pnpm clean && esbuild ./src/* --outdir=./lib --platform=node",
+    "clean": "rm -fr ./lib"
   },
   "type": "module",
   "bin": {
-    "wstd": "./bin/cli.js"
+    "webstudio": "./bin/cli.sh"
   },
-  "files": [
-    "README.md",
-    "lib/*",
-    "bin/*",
-    "!*.test.*"
-  ],
-  "license": "MIT"
-}
+  "license": "MIT",
+  "devDependencies": {
+    "@types/node": "^20.3.1",
+    "esbuild": "^0.18.5"
+  },
+  "dependencies": {
+    "@webstudio-is/asset-uploader": "^0.75.0",
+    "@webstudio-is/project-build": "^0.75.0",
+    "@webstudio-is/react-sdk": "^0.75.0",
+    "@webstudio-is/sdk-components-react": "^0.75.0",
+    "@webstudio-is/sdk-components-react-remix": "^0.75.0",
+    "deepmerge-ts": "^5.1.0",
```

Comment:
@TrySound any opinion on this dependency?

Reply:
I'd say we do not need default package.json. Just copy paste those 2 deps in every template.